app/components/employeeCard.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import styles from "./employeeCard.module.scss";
+
+export default function EmployeeCard(props: {
```

Comment:
What's the general practice of naming in Variant? I'm used to cards being interactive, but I don't see any onClick here?

Reply:
Cards will now have a href